frontend/packages/dev-console/src/components/import/icon/IconDropdown.scss

```diff
@@ -0,0 +1,20 @@
+.icon-dropdown {
+  img {
+    margin-right: var(--pf-global--spacer--sm);
+  }
+
+  .pf-c-dropdown__toggle-text {
+    display: flex;
+    align-items: center;
+  }
```

Comment:
```suggestion
```

Reply:
Without this lines it looks like this:

![image](https://user-images.githubusercontent.com/139310/88822336-02439480-d1c4-11ea-9fcf-4130c2c8ba7e.png)

And with this vertical alignment it looks like this:

![image](https://user-images.githubusercontent.com/139310/88822388-14253780-d1c4-11ea-9e7a-02f1bd9aafb0.png)

So I think we need then another vertical alignment (maybe a PatternFly css class I do not know yet) or keep this styles?

Reply:
Ah yeah... my bad. I didn't see it when I was scanning. That does look better. Let's keep it.

Reply:
Wait, I thought this was on the other one... this is not a patternfly component right? This shouldn't impact the styles...

Reply:
Yeah, this should be deleted, it's unused styles.

Reply:
Both styles are used. One for the list placeholder and one for the list items. But I will create one comma separated entry for both!